### DISCUSSION TITLE
Identify/select objects in trame/paraview window by mouse click

### DISCUSSION THREAD

**Discussion by thsmit** [Kitware/trame#648] [Trame/ParaView]
Hi All,

I am building a trame/paraview app. The app visualizes several objects in the trame/paraview window. I would like to be able to identify/select one object by mouse click. How is something like this possible? Show I implement some kind of 'pipeline browser' like in paraview?

Thanks in advance.

Best, Thijs

**Comment by jourdain** [comment]
While possible it would be hard to go over the details on a discussion thread.

If you want [we can work](https://www.kitware.com/trame/support/) with you to build such capability in your application.
